rfc_samplers.md

```diff
@@ -0,0 +1,125 @@
+
+Feature space
+-------------
+
+At a high level, a sampler returns a sequence of "clips" where a clip is a batch
+of (non-necessarily contiguous) frames. Based on an analysis of the sampling
+options of other libraries (torchvision, torchmultimodal, and various internal
+libraries), we want to support the following use-cases:
+
+
+**Clip location** (and number thereof)
+
+- Get `num_clips_per_video` random clips: typically called "Random" sampler
+- Get `num_clips_per_video` equally-spaced clips: "Uniform" sampler. Space is
```

Comment:
Can we add an option to get `num_clips_per_video` using a list of (start, end) timestamps?

Reply:
@arjish, the description here is at a high level, and is not talking about the parameters. Rather, it's talking about the sampling strategy. Can you take a look at the function signatures below to see if they fit your use cases? (It may be easier to read if you switch to "Display the rich diff") as that will hide our inline comments.)